x/bank/internal/invariants/invariants.go

```diff
@@ -1,12 +1,12 @@
-package bank
+package invariants
```

Comment:
shouldn't this be under `keeper` as other modules have it?

Reply:
I'm fine with that

Reply:
^^

Reply:
import cycles - I'd keep this as it is

Reply:
why? it shouldn't have any

Reply:
maybe it's because it's using `auth.Keeper` and not an expected `AccountKeeper`?

Reply:
Possibly - will double check

Reply:
But I don't think that that is the reason - bank's internal types depend on each other

Reply:
The scope of this PR is not to introduce functional changes/refactoring. So please as long as it works just +1 :)

Reply:
I still think this should leave in keeper

Reply:
Is this possible @alessio? I don't mind helping out here!